stroyprombeton/models.py

```diff
@@ -96,23 +104,13 @@ def __str__(self):
 class Product(catalog.models.AbstractProduct, pages.models.PageMixin):
     objects = catalog.models.ProductManager()
 
-    name = models.CharField(max_length=255, db_index=True, verbose_name=_('name'))
```

Comment:
seems we should not remove name here

Reply:
These fields are duplicated below, so I removed them